ocs_ci/deployment/deployment.py

```diff
@@ -370,12 +386,42 @@ def subscribe_ocs(self):
         if subscription_plan_approval == "Manual":
             wait_for_install_plan_and_approve(self.namespace)
 
+    def get_arbiter_location(self):
+        """
+        Get arbiter mon location for storage cluster
+        """
+
+        nodes = ocp.OCP(kind="node").get().get("items", [])
+
+        worker_nodes_zones = {
+            node["metadata"]["labels"].get("failure-domain.beta.kubernetes.io/zone")
+            for node in nodes
+            if "node-role.kubernetes.io/worker" in node["metadata"]["labels"]
+            and str(constants.OPERATOR_NODE_LABEL)[:-3] in node["metadata"]["labels"]
+        }
+
+        master_nodes_zones = {
+            node["metadata"]["labels"].get("failure-domain.beta.kubernetes.io/zone")
+            for node in nodes
+            if "node-role.kubernetes.io/master" in node["metadata"]["labels"]
```

Comment:
replace "node-role.kubernetes.io/master" with constants.MASTER_LABEL

Reply:
Done
In addition, changed the worker label also.

Reply:
thanks!

Reply:
Welcome